libs/pages/clusters/src/lib/ui/container/container.tsx

```diff
@@ -0,0 +1,13 @@
+import { ReactNode } from 'react'
+
+export interface ContainerProps {
+  children: ReactNode
+}
+
+export function Container(props: ContainerProps) {
+  const { children } = props
+
+  return <div className="bg-white flex rounded w-full min-h-[calc(100vh-10px)]">{children}</div>
```

Comment:
Maybe not rounding bottom corners? wdyt?
![CleanShot 2022-12-19 at 16 33 13@2x](https://user-images.githubusercontent.com/6163954/208462152-3e4a58d6-732c-4823-91ed-94c947f70d76.jpg)